src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -485,15 +484,9 @@ namespace Akka.Actor
         protected override bool SpecialHandle(object message, Akka.Actor.IActorRef sender) { }
         protected override void TellInternal(object message, Akka.Actor.IActorRef sender) { }
     }
-    public class DeadLetterMailbox : Akka.Dispatch.Mailbox
+    public sealed class DeadLetterMailbox : Akka.Dispatch.Mailbox
```

Comment:
marking class as `sealed`. Modifying this class could break DeathWatch and other capabilities.